signer/local/local.go

```diff
@@ -387,12 +387,18 @@ func (s *Signer) Sign(req signer.SignRequest) (cert []byte, err error) {
 		return nil, err
 	}
 
+	// Get the AKI from signedCert.  This is required to support Go 1.9+.
+	// In prior versions of Go, x509.CreateCertificate updated the
+	// AuthorityKeyId of certTBS.
+	block, _ = pem.Decode([]byte(signedCert))
+	parsedCert, _ := x509.ParseCertificate(block.Bytes)
```

Comment:
you can do parsedCert, _ := helpers.ParseCertificatePEM(signedCert)
https://godoc.org/github.com/cloudflare/cfssl/helpers#ParseCertificatePEM

Reply:
Cool.  Missed that.  Will modify